components/patient-data/migrations/src/main/java/org/phenotips/data/internal/R54691PhenoTips1378DataMigration.java

```diff
@@ -0,0 +1,143 @@
+/*
```

Comment:
Move this file in components/patient-access-rules/migrations/src/main/java/org/phenotips/data/permissions/internal